packages/devp2p/README.md

```diff
@@ -217,20 +241,29 @@ Send the initial status message with `sendStatus()`, then wait for the correspon
 to arrive to start the communication.
 
 ```ts
-eth.events.once('status', () => {
-  // Send an initial message
-  eth.sendMessage()
+// ./examples/peer-communication.ts#L96-L106
+
+eth.sendStatus({
+  td: intToBytes(17179869184), // total difficulty in genesis block
+  bestHash: hexToBytes('0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3'),
+  genesisHash: hexToBytes('0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3'),
 })
+
+// check CHECK_BLOCK
+let forkDrop: NodeJS.Timeout
+let forkVerified = false
+eth.events.once('status', () => {
+  eth.sendMessage(devp2p.ETH.MESSAGE_CODES.GET_BLOCK_HEADERS, [
 ```
 
 Wait for follow-up messages to arrive, send your responses.
 
 ```ts
-eth.events.on('message', async (code, payload) => {
-  if (code === devp2p.ETH.MESSAGE_CODES.NEW_BLOCK_HASHES) {
-    // Do something with your new block hashes :-)
-  }
-})
+// ./examples/peer-communication.ts#L116-L118
+
+eth.events.on('message', async (code: ETH.MESSAGE_CODES, payload: any) => {
+  if (code in ETH.MESSAGE_CODES) {
+    requests.msgTypes[code] = code + 1
```

Comment:
If a brief comment about what is happening and why in this line where `code` is being incremented and reassigned in `msgTypes` it would clarify what's happening in response to the message as part of the example.